ambari-agent/conf/unix/ambari-agent

```diff
@@ -62,7 +62,7 @@ export PATH=/usr/sbin:/sbin:/usr/lib/ambari-server/*:$PATH
 export AMBARI_CONF_DIR=$HOME_DIR/etc/ambari-server/conf:$PATH
 
 # Because Ambari rpm unpacks modules here on all systems
-export PYTHONPATH=/usr/lib/ambari-agent/lib:${PYTHONPATH:-}
+export PYTHONPATH=/usr/lib/ambari-agent/lib:/usr/lib/mpack-instance-manager:${PYTHONPATH:-}
```

Comment:
When will mpack-instance-manager be installed? Would we make ambari-agent rpm dependent on mpack-instance-manager rpm?

Reply:
For now the agent doesn't depend on the instance manager so you need to install it as a separate rpm, but I think we might want to add the dependency in future. The pythonPath needs to be updated in order to use the instance manager module from agent scripts